assets/components/organisms/restauration/restauration.twig

```diff
@@ -66,19 +66,19 @@
             </td>
             <td>
               <div class="row">
-                <div class="col-6 text-primary">E</div>
+                <div class="col-6 text-primary"><abbr title="Étudiant">E</abbr></div>
                 <div class="col-6 text-right">11.90</div>
               </div>
               <div class="row">
-                <div class="col-6 text-primary">V</div>
+                <div class="col-6 text-primary"><abbr title="Vieux">V</abbr></div>
                 <div class="col-6 text-right">14.00</div>
               </div>
               <div class="row">
-                <div class="col-6 text-primary">D</div>
+                <div class="col-6 text-primary"><abbr title="Daltonien">D</abbr></div>
```

Comment:
Needs real (or plausible) wording before we merge this.

Reply:
I found the legend at the bottom of the page 😉
D -> Doctorants